.github/workflows/tests.yml

```diff
@@ -26,8 +26,8 @@ jobs:
   go-tidy:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-go@v2
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v3
```

Comment:
ditto

Reply:
I prefer not caching for the static check because it's fast (not a bottleneck) and re-downloading the dependencies could be part of the check process.

Reply:
SG.